src/widgets/contact-details/relationships.resource.tsx

```diff
@@ -2,11 +2,27 @@ import { openmrsFetch } from "@openmrs/esm-api";
 
 export function fetchPatientRelationships(patientIdentifier: string) {
   return openmrsFetch<{ results: Relationship[] }>(
-    `/ws/rest/v1/relationship?person=${patientIdentifier}`
+    `/ws/rest/v1/relationship?v=full&person=${patientIdentifier}`
```

Comment:
@denniskigen this will likely return a lot of unneeded data. We can merge for now but I would suggest using a more custom request than full.

Reply:
@jonathandick fixed in #155